Source/MipsExecutor.cpp

```diff
@@ -363,14 +361,13 @@ void CMipsExecutor::PartitionFunction(uint32 functionAddress)
 	//Create blocks
 	{
 		uint32 currentPoint = -1;
-		for(PartitionPointSet::const_iterator pointIterator(partitionPoints.begin());
-			pointIterator != partitionPoints.end(); pointIterator++)
+		for(auto partitionPoint : partitionPoints)

```

Comment:
@jpd002 I just want you to confirm if my naming choice(`partitionPoint`)  is correct here?